docs/userguide/containers/dockerizing.md

```diff
@@ -24,147 +24,131 @@ Running an application inside a container takes a single command: `docker run`.
 
 ## Run a Hello world
 
-Let's try it now.
+Let's run a hello world container.
```

Comment:
I kind of question the purpose of this entire piece, so I'm not sure about the effort we are putting into it.  It is a standalone piece and not really part of the "learn by example flow."  WDYT Lynda?

Also, the piece switches between first and second person, we should keep it to second:

![image](https://cloud.githubusercontent.com/assets/1347209/13750351/55ae8402-e9c2-11e5-9909-c6911c170a0b.png)